src/protocol/templates/ping_doc.in

```diff
@@ -28,6 +28,10 @@
 | {{field.index}} | {{field.type}} | {{field.name}} | {{field.description}} |
 {% endfor %}
 {% endif %}
+{% endfor %}
```

Comment:
This patch only adds a newline to the output. I don't understand the need/reasoning (the same 'print format once' logic is already accomplished on L26.

Probably, we should change L25 to
`{% set messages = sensors[**'Common'**]['messages'] %}`
And remove the loop at L24.

```diff
$ git diff README.md
diff --git a/README.md b/README.md
index acc527a..32dea40 100644
--- a/README.md
+++ b/README.md
@@ -74,6 +74,7 @@
 | 7 | u8 | dst_device_id | The device ID of the intended recipient of the message. |
 | 8-n | u8[] | payload | The message payload. |
 | (n+1)-(n+2) | u16 | checksum | The message checksum. The checksum is calculated as the sum of all the non-checksum bytes in the message. |
+
 ## Common Messages
 
 ### general
```